aviary/examples/run_reserve_mission_multiphase_time_and_range.py

```diff
@@ -0,0 +1,179 @@
+            "initial_mach": (0.36, "unitless"),
+            "final_mach": (0.72, "unitless"),
+            "mach_bounds": ((0.18, 0.74), "unitless"),
+            "initial_altitude": (0.0, "ft"),
+            "final_altitude": (32000.0, "ft"),
+            "altitude_bounds": ((0.0, 32000.0), "ft"),
+            "throttle_enforcement": "path_constraint",
+            "fix_initial": True,
+            "constrain_final": False,
+            "fix_duration": False,
+            "initial_bounds": ((0.0, 0.0), "min"),
+            "duration_bounds": ((64.0, 192.0), "min"),
+        },
+        "initial_guesses": {"times": ([0, 128], "min")},
+    },
+    "reserve_cruise_fixed_range": {
+        "subsystem_options": {"core_aerodynamics": {"method": "computed"}},
+        "user_options": {
+            "reserve": True,
+            # Distance traveled in this phase
+            "target_distance": (200, 'km'),
+            "optimize_mach": False,
+            "optimize_altitude": False,
+            "polynomial_control_order": 1,
+            "num_segments": 5,
+            "order": 3,
+            "solve_for_distance": False,
+            "initial_mach": (0.72, "unitless"),
+            "final_mach": (0.72, "unitless"),
+            "mach_bounds": ((0.7, 0.74), "unitless"),
+            "initial_altitude": (32000.0, "ft"),
+            "final_altitude": (32000.0, "ft"),
+            "altitude_bounds": ((23000.0, 38000.0), "ft"),
+            "throttle_enforcement": "boundary_constraint",
+            "fix_initial": False,
+            "constrain_final": False,
+            "fix_duration": False,
+            "initial_bounds": ((149.5, 448.5), "min"),
+            "duration_bounds": ((0, 300), "min"),
+        },
+        "initial_guesses": {"times": ([30, 120], "min")},
+    },
+    "reserve_cruise_fixed_time": {
+        "subsystem_options": {"core_aerodynamics": {"method": "computed"}},
+        "user_options": {
+            "reserve": True,
+            # Time length of this phase
+            "target_time": (45, "min"),
+            "optimize_mach": False,
+            "optimize_altitude": False,
+            "polynomial_control_order": 1,
+            "num_segments": 5,
+            "order": 3,
+            "solve_for_distance": False,
+            "initial_mach": (0.72, "unitless"),
+            "final_mach": (0.72, "unitless"),
+            "mach_bounds": ((0.7, 0.74), "unitless"),
+            "initial_altitude": (32000.0, "ft"),
+            "final_altitude": (32000.0, "ft"),
+            "altitude_bounds": ((23000.0, 38000.0), "ft"),
+            "throttle_enforcement": "boundary_constraint",
+            "fix_initial": False,
+            "constrain_final": False,
+            "fix_duration": True,  # TBD remove in future
+            "initial_bounds": ((149.5, 448.5), "min"),
+            "duration_bounds": ((0, 0), 'min'),  # TBD remove in future
+        },
+    },
+    "reserve_cruise_fixed_range_2": {
```

Comment:
I really like how this demonstrates the arbitrary ordering of reserve phases that is now possible.